rest_framework/routers.py

```diff
@@ -291,6 +291,7 @@ def get(self, request, *args, **kwargs):
                     try:
                         ret[key] = reverse(
                             url_name,
+                            kwargs=kwargs,
```

Comment:
Yes we should, I just didn't have any use case of not named parameters (as I'm not used to using it).